src/rust/engine/process_execution/src/remote_tests.rs

```diff
@@ -1436,7 +1436,10 @@ async fn initial_response_error() {
     .await
     .expect_err("Want Err");
 
-  assert_eq!(result.to_string(), "Internal: Something went wrong");
+  assert_eq!(
+    result.to_string(),
+    "Error from remote execution: InvalidArgument: \"Execute endpoint called. Did not expect this call.\""
```

Comment:
Nit: This appears to be because the retry results in more methods being called on the mock... might be clearer to add more mock responses to actually end the session cleanly?